modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -889,6 +898,9 @@ public List<File> generate() {
             }
         }
 
+        configureGeneratorProperties();
+        configureOpenAPIInfo();
+
```

Comment:
How about removing the InlineModelResolver code from this fn and keeping it where you added it in configureGeneratorProperties?

Reply:
Sure thing!